src/main/java/websocket/models/Request.java

```diff
@@ -39,6 +44,34 @@
     @JsonIgnore
     private IRequestSendErrorHandler errorHandler;
 
+    /**
+     * Default Request constructor that should only be used for testing requests.
+     */
+    public Request() {
+    	this.instance = this;
+    	this.tag = tagGenerator.getAndIncrement();
+        this.resource = "Default";
+        this.method = "Default";
+        // TODO: get these from the client core based on user info
+        this.senderId = "12345";
+        this.senderToken = "12345";
+        this.timestamp = System.currentTimeMillis();
```

Comment:
when this object goes to json, are they fields still lowercase? that might be an issue, because they start with an uppercase letter on the server